packages/editor-ui/src/components/ValueSurvey.vue

```diff
@@ -9,55 +9,71 @@
 		class="value-survey"
 	>
 		<template slot="header">
-			<span :class="$style.title" v-html="getTitle" />
+			<div :class="$style.title">
+				<n8n-heading tag="h2" size="medium" color="text-xlight">{{ getTitle }}</n8n-heading>
+			</div>
 		</template>
 		<template slot="content">
 			<section :class="$style.content">
-				<div v-if="showButtons" :class="$style.buttons">
-					<n8n-button @click="selectSurveyValue('very')" label="Very disappointed" />
-					<n8n-button @click="selectSurveyValue('somewhat')" label="Somewhat disappointed" />
-					<n8n-button @click="selectSurveyValue('not')" label="Not disappointed (it isn’t really that useful)" />
+				<div v-if="showButtons" :class="$style.wrapper">
+					<div :class="$style.buttons">
+						<div :class="$style.button" v-for="value in 11" :key="value - 1" >
+							<n8n-button
```

Comment:
ux change - talked to max about this.. these buttons need a hover state. can you make the text color orange on hover?

Reply:
Done